doc/api/n-api.md

```diff
@@ -6592,6 +6592,145 @@ idempotent.
 
 This API may only be called from the main thread.
 
+## Using embedded Node.js
```

Comment:
I think I'd leave the additions in api/n-api.md to be in embedding.md instead to avoid any confusion that there is a promise of ABI stability.

Reply:
Agree. I did not touch the docs yet.

Reply:
The latest commit removes all changes from the n-api.md.